core/services/workflows/cmd/cre/README.md

```diff
@@ -6,6 +6,38 @@ There are two step to executing the engine in stand alone mode:
 1. Compile the workflow from source
 2. Run the engine with the compiled workflow binary
 
+## Installing Capability Binaries
+Install the capability binaries you need in a workflow via the core node make file scripts:
+
+```bash
+make install-loopinstall
+make install-plugins-public
+make install-plugins-private
```

Comment:
We call these three together in several places - why don't we make a single `make install-plugins` that triggers them all or something?